Rakefile

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env rake
 
+module TempFixForRakeLastComment
```

Comment:
My local Ruby system must be higher than the one our build is using because I can't compile with last_comment in place.  Is there a Ruby command I should be using to compile using libraries this code expects?